kouncil-frontend/libs/common-login/src/lib/login/common-login.component.ts

```diff
@@ -1,44 +1,54 @@
-import {ChangeDetectionStrategy, Component, EventEmitter, Output} from '@angular/core';
+import {ChangeDetectionStrategy, Component, EventEmitter, Input, Output} from '@angular/core';
 import {FormBuilder, FormControl, FormGroup, Validators} from "@angular/forms";
 import {User} from "./user";
+import {Backend} from "@app/common-model";
 
 @Component({
   selector: 'app-common-login',
   template: `
-    <div class="icon-login-container">
+    <div class="icon-login-container"
+         [ngClass]="backend === 'SERVER' ? 'icon-login-container-desktop' : 'icon-login-container-demo'">
```

Comment:
Can you use composition? Use <ng-content slots> to pass this, instead of implementing that here.